src/components/NavigationBar/NavigationBar.module.css

```diff
@@ -0,0 +1,67 @@
+.root {
+    display: flex;
+    flex-direction: column;
+    min-height: 100vh;
```

Comment:
can u remove this? seems to be causing the components to get pushed down

Reply:
seems to be resolved in #50